dp_mobility_report/model/preprocessing.py

```diff
@@ -44,12 +44,13 @@ def _validate_columns(df: pd.DataFrame) -> pd.DataFrame:
         raise ValueError("Column 'lng' must be present in data.")
     if not pd.core.dtypes.common.is_float_dtype(df[const.LNG]):
         raise TypeError("Column 'lng' is not of type float.")
-    if const.DATETIME not in df.columns:
+    if const.DATETIME not in df.columns and timestamps:
```

Comment:
maybe it would be more elegant, if we dont set the parameter "datetime", instead get that info from the dataset. If there is no DATETIME column, we show an info to the user "there is no datetime column in the data. Reports with temporal information will be omitted." Then we do not need to raise in error here and we don't need the additional parameter "datetime" (maybe there is a reason why its still useful?)